lib/ht_item_overlap.rb

```diff
@@ -16,11 +16,13 @@ def initialize(ht_item)
 
   # Find all organization with holdings that match the given ht_item
   def organizations_with_holdings
```

Comment:
I do not understand this.

Reply:
Consider extracting conditions as methods with intention-revealing names?